ipa-core/src/bin/report_collector.rs

```diff
@@ -534,7 +534,7 @@ async fn ipa_test(
         r
     };
 
-    let mut key_registries = KeyRegistries::default();
+    let key_registries = KeyRegistries::default();
     let Some(key_registries) = key_registries.init_from(network) else {
```

Comment:
IIRC, the `mut` was needed to call `init_from`? but if this is compiling that must not be needed.

Reply:
yea I changed the signature of this method as it is always used to fully consume `KeyRegistries`